leaderboards/test_services.py

```diff
@@ -37,7 +37,7 @@ def test_nochoke_leaderboard(self, stub_user_stats, score_filter):
         )
         membership = leaderboard.memberships.first()
         assert membership.score_count == 5
-        assert membership.pp == 1294.9049944518279
+        assert membership.pp == 1296.314845169917
```

Comment:
increasing because previously difficalcy calculated acc without slider tails and ticks, so there was a slight pp loss for lazer scores.